EIPS/7639.md

```diff
@@ -0,0 +1,81 @@
+---
+eip: 7639
+title: Cease serving history before PoS
+description: Execution layer clients will no longer serve block data before Paris over p2p.
+author: lightclient (@lightclient)
+discussions-to: https://ethereum-magicians.org/t/cease-serving-history-before-pos/18991
+status: Draft
+type: Standards Track
+category: Networking
+created: 2024-02-13
+---
+
+## Abstract
+
+Execution layer clients will no longer request or respond to p2p queries about
+block data before the Paris upgrade.
+
+## Motivation
+
+As of 2024, historical data in clients has grown to around 500 GB. Nearly 400 GB
+of that is from block data before PoS was activated in the Paris upgrade. Long
+term, Ethereum plans to bound the amount of data nodes must store. This EIP
+proposes the first steps to achieve such goal.
+
+## Specification
+
+Clients must not make or respond to p2p queries about blocks before block 15537393.
+
+### Header Accumulator
```

Comment:
This feels out of context. even though i know what this is, it does not seem connected to the rest of the EIP. maybe we need to add some explaination about how the header accumulator will be used. and how users will be able to get this data using era1 one files?

Reply:
I think you're right. I have moved it to a separate EIP: https://github.com/ethereum/EIPs/pull/8277